checker/examples/MavenExample/pom.xml

```diff
@@ -92,7 +95,10 @@
               <compilerArgs>
                   <!-- location of the annotated JDK, which comes from a Maven dependency -->
                   <arg>-Xbootclasspath/p:${annotatedJdk}</arg>
-                  <arg>-J-Xbootclasspath/p:${typeAnnotationsJavac}</arg>
+                  <!--treat Checker Framework errors as warnings-->
```

Comment:
I see this is consistent with the previous behavior.  What is the rationale for this behavior?  I agree a user could set it if desired, but I don't see that we should be recommending that all Maven users always use -Awarns.  Therefore, I would omit this.